net/cloudflared/files/sample_config.yml

```diff
@@ -0,0 +1,3 @@
+url: http://localhost:80
```

Comment:
```suggestion
url: http://localhost:8000
```

To match Cloudflare's documentation, and to not accidentally tunnel luci.

Reply:
Thank you for your suggestion!